aar_doc/core.py

```diff
@@ -21,6 +22,24 @@
 yaml.allow_unicode = True
 
 
+def ansible_doc_markup(text):
```

Comment:
This is a great idea!

I would like to see tests for each element that can be formatet in our template. Not every formatting needs to be checked, but we should make sure that all elements that use this filter also actually use it.

Reply:
Added test cases. I think I covered all of the uses.